tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/TestScriptTests/PropertyMock/propertiesluis.mock.dialog

```diff
@@ -0,0 +1,18 @@
+﻿{
+    "$schema": "../../../../tests.schema",
+    "$kind": "Microsoft.Test.PropertiesMock",
+    "assignments": [
+        {
+            "property": "settings.luis.lu",
+            "value": "00000000-0000-0000-0000-000000000000"
+        },
+        {
+            "property": "settings.luis.endpointKey",
+            "value": "00000000000000000000000000000000"
+        },
+        {
+            "property": "settings.luis.endpoint",
+            "value": "https://westus.api.cognitive.microsoft.com"
+        }
+    ]
+}
```

Comment:
This works, but it is a little different than the normal LUIS flow, i.e. you do luis:build and get a settings file.  I see you are caching the LUIS responses, but now I need to manually copy the settings into a file like this and run once to cache and once not--this makes it much more complicated to do the LUIS mocking.  I don't think you technically need to do that--can't you still use the regular properties file?

Reply:
Updated.